.gitignore

```diff
@@ -16,3 +16,9 @@
 .byebug_history
 
 /config/master.key
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
```

Comment:
Please look at line 18  

Reply:
Just saw this and fixed the issue, should have seen that apologies.